modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLTranslator.java

```diff
@@ -0,0 +1,184 @@
+package com.liferay.translation.translator.deepl.internal.translator;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.petra.string.StringUtil;
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+import com.liferay.portal.kernel.module.configuration.ConfigurationProvider;
+import com.liferay.portal.kernel.util.Validator;
+import com.liferay.translation.translator.Translator;
+import com.liferay.translation.translator.TranslatorPacket;
+import com.liferay.translation.translator.deepl.internal.configuration.DeepLTranslatorConfiguration;
+import com.liferay.translation.translator.deepl.internal.constants.DeepLConstants;
+import com.liferay.translation.translator.deepl.internal.model.SupportedLanguage;
+import com.liferay.translation.translator.deepl.internal.model.TranslateResponse;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Reference;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+@Component(
+    configurationPid = "com.liferay.translation.translator.deepl.internal.configuration.DeepLTranslatorConfiguration",
+    service = Translator.class
+)
+public class DeepLTranslator implements Translator {
+  @Override
+  public boolean isEnabled(long companyId) throws ConfigurationException {
+    DeepLTranslatorConfiguration
+        deepLTranslatorConfiguration =
+        _configurationProvider.getCompanyConfiguration(
+            DeepLTranslatorConfiguration.class, companyId);
+
+    return deepLTranslatorConfiguration.enabled() &&
+        !Validator.isBlank(deepLTranslatorConfiguration.authKey()) &&
+        !Validator.isBlank(deepLTranslatorConfiguration.url());
```

Comment:
This kind of inline expression, though correct, are not allowed by our code conventions. You need to write an explicit conditional:
```
if (deepLTransator....) {
    return true;
}

return false;
```